visual_diff.py

```diff
@@ -138,10 +142,15 @@ def make_matrix(tokens_a, tokens_b):
         data_b = get_tokens(f_b.read(), language)
     matrix = make_matrix(data_a.tokens, data_b.tokens)
 
+    if args.color:
+        hues = find_duplicates.get_hues(matrix, args.filename_b is None)
+    else:
+        hues = None
+
     if args.output_location is None:
         if can_use_gui:
             text_width = get_text_width(args)
-            gui.launch(matrix, data_a, data_b, args.map_width, text_width)
+            gui.launch(matrix, data_a, data_b, args.map_width, text_width, hues)
```

Comment:
Now that we're always passing in `hues`, the arguments elsewhere should no longer be optional.

Reply:
...and reorder the args to make more sense once they're required. `hues` should be immediately after `matrix`.